versioned_docs/version-22.04/releases/centreon-commercial-extensions.md

```diff
@@ -16,6 +16,15 @@ If you have feature requests or want to report a bug, please contact support.
 
 ## Centreon MAP
 
+### 22.04.10
+
+Release date: `September 18, 2023`
+
+#### Bug fixes
+
+- [Packaging] Fixed a regression in centreon-map-server and centreon-map-engine packaging that caused a broken dependency situation with Java.
```

Comment:
As there are 2 different chapters between Engine and Legacy, I will only mention the Engine package here.